docs/scripting-ref.rst

```diff
@@ -331,6 +331,32 @@ Compare:
 See also: :ref:`splash-runjs`, :ref:`splash-jsfunc`,
 :ref:`splash-wait-for-resume`, :ref:`splash-autoload`.
 
+.. _splash-disablejs:
+
+splash:disablejs
```

Comment:
Do we need separate `disable`/`enable` funcs? There's already a `splash:set_images_enabled`, so `set_js_enabled(true/false)` seems more natural.